### ISSUE TITLE
DetailsList: Duplicate narration with groups attribute on focus leave

### ISSUE THREAD

**Issue by callmedhh** [microsoft/fluentui#11621]
### Environment Information

- **Package version(s)**: Fabric react 7
- **Browser and OS versions**: Chrome and Edge.

### Describe the issue: 
Assume there is a DetailsList component that has groups attribute. Turn on narrator and focus on one row in the list. Press tab to let focus goes out of DetailsList. Screen reader will read "Exit Table" twice. Note that if we do not set groups in DetailsList, screen reader will only read "Exit Table" once.

#### Actual behavior:

Screen reader reads Exit table twice.

#### Expected behavior:

Screen reader reads Exit table once.

**Comment by JasonGore** [comment]
Thanks for the example, it shows the issue nicely. To clarify, Narrator reads "Exit Table" when focus moves from an item within a group to exit the `DetailsList` entirely.

Based on #7063 the current design of `GroupedList` uses ARIA role `grid`. Since `DetailsList` uses `GroupedList` and also uses `grid` for its container, we end up with nested `grids`. As a result, any given group item is within nested grids. As a result, when removing focus from a group item to outside the `DetailsList`, the nested table situation results in "Exit Table" being narrated twice.

Unfortunately I don't see a way to override this behavior. Additionally, I don't think the fix will be easy as it requires examining the existing accessibility approach taken by `DetailsList` and/or `GroupedList`. Should instead use a [`role` of `group`](https://www.w3.org/TR/wai-aria-1.1/#group) by default, [rather than `grid`](https://github.com/OfficeDev/office-ui-fabric-react/blob/bb72494fc5de1273568f80d8f5b814825ff3c510/packages/office-ui-fabric-react/src/components/GroupedList/GroupedListSection.tsx#L317)? And should such a change apply to both `GroupedList` and `DetailsList`? These questions will require more investigation and consulting with partners to determine the best path moving foward.

**Comment by xugao** [comment]
@callmedhh - I'm actually not sure if this is a bug. in [this example](https://codepen.io/callmedhh/pen/NWPXZEp), both when user enters the DetailList and enters Group1, Narrators announces "Table". So user should know there is table inside table. So when it exits, doesn't it make sense to announce "Exit" twice?  but regardless, aria guidance didn't say we cannot have nested grid. so if anything incorrect, it probably requires fix on the Narrator side.

**Comment by callmedhh** [comment]
> @callmedhh - I'm actually not sure if this is a bug. in [this example](https://codepen.io/callmedhh/pen/NWPXZEp), both when user enters the DetailList and enters Group1, Narrators announces "Table". So user should know there is table inside table. So when it exits, doesn't it make sense to announce "Exit" twice? but regardless, aria guidance didn't say we cannot have nested grid. so if anything incorrect, it probably requires fix on the Narrator side.

Hi, I think we should only announce "Exit" once, as you can try in GroupedList, it is also table inside table, but in that case, screen reader will only announce "Exit" once.

**Comment by xugao** [comment]
@callmedhh - could you pls elaborate on the GroupList case and how/if it enters table twice but exists once? Do you mean this example: https://developer.microsoft.com/en-us/fabric#/controls/web/groupedlist?

**Comment by callmedhh** [comment]
> @callmedhh - could you pls elaborate on the GroupList case and how/if it enters table twice but exists once? Do you mean this example: https://developer.microsoft.com/en-us/fabric#/controls/web/groupedlist?

Yes, I mean this example.

**Comment by xugao** [comment]
@callmedhh - could you tell me why you think GroupList is a table-within-table case?
If you hear the screen reader "enters table" twice, could you give me the step-by-step repro instructions?

Thank you for your patience and sorry in advance if i missed something obvious

**Comment by callmedhh** [comment]
> @callmedhh - could you tell me why you think GroupList is a table-within-table case?
> If you hear the screen reader "enters table" twice, could you give me the step-by-step repro instructions?
> 
> Thank you for your patience and sorry in advance if i missed something obvious

The reason I call GroupedList is also a table-within-table case is that you call the DetailsList example I give to you as table-in-table. And GroupedList looks exactly the same as the DetailsList with groups attributes.

**Comment by xugao** [comment]
@callmedhh - i get your reasonings. will look into ways to address this.

**Comment by callmedhh** [comment]
> @callmedhh - i get your reasonings. will look into ways to address this.

Thank you a lot for your patience, can you give us an ETA for this bug?

**Comment by xugao** [comment]
@callmedhh - sorry for the delayed response. i am still looking into what the potential fix is. I don't have an ETA yet but will keep you posted
